### DISCUSSION TITLE
can i join team?

### DISCUSSION THREAD

**Discussion by putraaxyzo** [ROM-builders/temporary#10277] [Q&A] [answered]
my build :
https://cirrus-ci.com/build/6608882163253248
my tree : 
https://github.com/putraaxyzo/device_xiaomi_juice
currently maintaining :
Raven-OS [OFFICIAL]
ProjectBlaze [OFFICIAL]

**Comment by Apon77** [comment, marked as answer]
> my build : https://cirrus-ci.com/build/6608882163253248 my tree : https://github.com/putraaxyzo/device_xiaomi_juice currently maintaining : Raven-OS [OFFICIAL] ProjectBlaze [OFFICIAL]

give telegram id

**Reply by putraaxyzo** [reply]
@putrazxyo13
